### PR TITLE
VP-6889: Email address verification template management

### PR DESCRIPTION
**Pull request by AliveMen**
Register Email address verification template

### CODE REVIEW

**Review by Vectorfield4 — APPROVED** [review]

File: src/VirtoCommerce.NotificationsModule.Core/Types/VerificationEmailNotification.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+using VirtoCommerce.NotificationsModule.Core.Model;
+
+namespace VirtoCommerce.NotificationsModule.Core.Types
+{
+    public class VerificationEmailNotification : EmailNotification
```
Comment (eMazeika) [inline src/VirtoCommerce.NotificationsModule.Core/Types/VerificationEmailNotification.cs:5]: add short class description

File: src/VirtoCommerce.NotificationsModule.Core/Types/VerificationEmailNotification.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    {
+        public VerificationEmailNotification() : base(nameof(VerificationEmailNotification))
+        {
+            //for backward compatibility v.2
+            Alias = "EmailVerificationNotification";
```
Comment (eMazeika) [inline src/VirtoCommerce.NotificationsModule.Core/Types/VerificationEmailNotification.cs:10]: does new feature have/need any backward compatibility? Maybe it's not needed?

File: src/VirtoCommerce.NotificationsModule.Core/Types/VerificationEmailNotification.cs [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        }
+
+        public VerificationEmailNotification(string type) : base(type)
+        {
+
+        }
+
+        public string Url { get; set; }
+
```
Comment (ksavosteev) [inline src/VirtoCommerce.NotificationsModule.Core/Types/VerificationEmailNotification.cs:19]: remove empty line

File: src/VirtoCommerce.NotificationsModule.Core/Types/VerificationEmailNotification.cs [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    }
+
```
Comment (ksavosteev) [inline src/VirtoCommerce.NotificationsModule.Core/Types/VerificationEmailNotification.cs:21]: remove empty line

**Comment by eMazeika** [comment]
@Vectorfield4 , don't we have this template already? https://github.com/VirtoCommerce/vc-module-notification/blob/4524cf9d3cba193f27bb7fc89ccf207177b779cd/src/VirtoCommerce.NotificationsModule.Core/Types/ConfirmationEmailNotification.cs

File: src/VirtoCommerce.NotificationsModule.Core/Types/VerificationEmailNotification.cs (hunk 1 above)
Comment (AliveMen) [reply src/VirtoCommerce.NotificationsModule.Core/Types/VerificationEmailNotification.cs:9]: Added

File: src/VirtoCommerce.NotificationsModule.Core/Types/VerificationEmailNotification.cs (hunk 2 above)
Comment (AliveMen) [reply src/VirtoCommerce.NotificationsModule.Core/Types/VerificationEmailNotification.cs]: Removed

File: src/VirtoCommerce.NotificationsModule.Core/Types/VerificationEmailNotification.cs (hunk 4 above)
Comment (AliveMen) [reply src/VirtoCommerce.NotificationsModule.Core/Types/VerificationEmailNotification.cs]: Done

File: src/VirtoCommerce.NotificationsModule.Core/Types/VerificationEmailNotification.cs (hunk 3 above)
Comment (AliveMen) [reply src/VirtoCommerce.NotificationsModule.Core/Types/VerificationEmailNotification.cs]: removed

**Review by akak1977 — APPROVED** [review]

**Pull request merged**